lib/product-import.coffee

```diff
@@ -168,7 +168,6 @@ class ProductImport
         @client.productProjections
         .where(predicate)
         .staged(true)
-        .perPage(100)
```

Comment:
![Seems legit](http://s2.quickmeme.com/img/7b/7b703c294fa76e39670104cf55def8aa32b59629976c1f016e53e1fa732de50d.jpg)
I will update it in a second